misc/xschem/meta.yaml

```diff
@@ -0,0 +1,61 @@
+# Use `conda-build-prepare` before building for a better version string.
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG, GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+{% set xpm_version = '3.5.14' %}
+{% set xpm_md5 = 'c308d303ed5d369a8c65b09f83a2f86b' %}
+
+package:
+  name: xschem
+  version: {{ version }}
+
+source:
+   - git_url: https://github.com/StefanSchippers/xschem.git
+     git_rev: master
+   - url: https://www.x.org/releases/individual/lib/libXpm-{{ xpm_version }}.tar.gz
+     md5: {{ xpm_md5 }}
+     folder: xpm
+
+build:
+  # number: 201803050325
+  number: {{ environ.get('DATE_NUM') }}
+  # string: 20180305_0325
+  string: {{ environ.get('DATE_STR') }}
+  script_env:
+    - CI
+    - CONDA_OUT
+    - TEST_PACKAGE
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - make
+    - bison
+    - flex
+    - {{ compiler('c') }}
+    - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
```

Comment:
These `cdt` dependencies are required to run the program, right?

I'm pretty sure that `cdt` dependencies are not auto-detected / checked and hence have to be manually specified?

Reply:
yep, my understanding is that those don't need to be added to `run` since they are part of the sysroot.

Reply:
and it's similar to what's being done for the magic, netgen and openroad packages:
https://github.com/hdl/conda-eda/blob/master/misc/netgen/meta.yaml#L24-L27
https://github.com/hdl/conda-eda/blob/master/misc/magic/meta.yaml#L29-L33

So if we want to start doing something different from now on, I'd rather unify all of those consistently.

Reply:
Yeap, make sense to me.